mava/systems/ippo/networks.py

```diff
@@ -256,24 +256,29 @@ def policy_fn(inputs: jnp.ndarray) -> networks_lib.FeedForwardNetwork:
             FeedForwardNetwork class
         """
         # Add the observation network and an MLP network.
-        policy_network = [
-            MLP_NORM(
-                policy_layer_sizes,
-                activation=activation_function,
-                w_init=w_init_fn(orthogonal_initialisation, jnp.sqrt(2)),
-                activate_final=True,
-                layer_norm=layer_norm,
-            ),
-        ]
+        policy_layers = []
+        if observation_network:
+            policy_layers.extend([observation_network])
+        policy_layers.extend(
```

Comment:
I think keep this consistent with below and append the item instead of extending a list of the item